pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -279,6 +279,7 @@
     private Boolean careEvolve;
     @ColumnName(DBConstants.UPLOAD_REASON)
     private String uploadReason;
+    private long ddpInstanceId;
```

Comment:
we need to add `@ColumnName` annotation to the field to make `ES` feature to export this field as well, also let's change primitive long type to reference long to avoid exporting any misleading data(by default long gets 0 as a value) to `ES`.